homeassistant/components/telegram_bot/__init__.py

```diff
@@ -219,11 +218,8 @@ def load_data(hass, url=None, filepath=None, username=None, password=None,
 
     p_type = p_config.get(CONF_PLATFORM)
 
-    platform = await async_prepare_setup_platform(
-        hass, config, DOMAIN, p_type)
-
-    if platform is None:
-        return
+    platform = importlib.import_module('.{}'.format(config[CONF_PLATFORM]),
```

Comment:
wrap it in executor?

Reply:
We don't do this with other imports. It's only in the beginning. I dunno 🤷‍♂️